sdk/src/main/scala/io/horizen/account/utils/AccountFeePaymentsUtils.scala

```diff
@@ -57,4 +57,21 @@ object AccountFeePaymentsUtils {
       }
     }
   }
+
+  def getMainchainWithdrawalEpochDistributionCap(epochMaxHeight: Long, epochLength: Long): BigInteger = {
+    val baseReward = 12.5 * 1e8
+    val halvingInterval = 840000
```

Comment:
This `halvingInterval` value would not be valid for regtest (there it is 2000), could it be useful to have a network dependent parameter?